.env.example

```
@@ -14,4 +14,12 @@ SERVER_PORT=8080
 SWAGGER_ENABLED=true
 
 CORS_ALLOWED_ORIGINS=http://localhost:3000
-CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
\ No newline at end of file
+CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
+
+S3_BUCKET=
+S3_REGION=
+S3_PUT_EXPIRATION=300000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_PUT_EXPIRATION key should go before the S3_REGION key

(UnorderedKey)